emailserver.js

```diff
@@ -45,21 +45,23 @@ var server = smtp.createServer(HOSTNAME, function (req) {
 
     mailparser.on('end', function(mail) {
       mail.receivedAt = new Date().toISOString();
-      var user = req.to;
-      log('Received message for', user);
+      var users = req.to;
+      log('Received message for', users);
```

Comment:
It turns out that `req.to` is always an array, even if there's only a single user.  So I'm not sure why this was working properly before, maybe some magic in the redis module?